src/OhmmsData/libxmldefs.h

```diff
@@ -128,8 +128,8 @@ inline bool putContent(std::vector<T>& a, const xmlNodePtr cur)
       b.push_back(t);
     else if (!stream.eof() && stream.fail())
     {
-      std::cerr << "failbit detected when reading type (type_info::name) "
-                << typeid(T).name() << ", value " << t << std::endl;
+      std::cerr << "Error parsing XML for type (type_info::name) " << typeid(T).name() << ", value " << t
```

Comment:
I assume this is the error which is going to become an exception?

Reply:
Yes